cost-analyzer/values.yaml

```diff
@@ -706,12 +706,24 @@ awsstore:
   useAwsStore: false
   createServiceAccount: false
 
+federatedETL:
+  federatedCluster: false
+  primaryCluster: false
+  redirectS3Backup: false
+  useExistingS3Config: false
+  federator:
+    # federationCutoffDate: "2022-10-18T00:00:00.000Z"
+    enabled: false
+    clusters: []
```

Comment:
Should this be marked as `# optional cluster_id whitelist` ?

Reply:
Yes, comment explains this!

Reply:
Is it cluster_id's or cluster_name's? Worth clarifying?